tests/serhiia/test_hw1.py

```diff
@@ -0,0 +1,30 @@
+
+
+def test_max_value():
+    x = max(3, 4, 5)
+    assert x == 5
+
+def test_min_value():
+    x = min([5, 3, 4])
+    assert x == 3
+
+def test_max_string():
+    x = max("sfdsdf")
+    assert x == 's'
+
+def test_min_value_by_length():
+    x = min("banana", "ball", "cat", key=len)
+    assert x == "cat"
+
+def test_true_or_false():
+    x = [] or "abc"
+    assert x == "abc"
+
+def test_true_or_false_blnk():
```

Comment:
try to avoid typos in function name


test_true_or_false_blnk. -> test_true_or_false_blank